articles/static-web-apps/publish-devops.md

```diff
@@ -29,7 +29,7 @@ In this tutorial, you learn to:
 ## Create a static web app in an Azure DevOps
 
   > [!NOTE]
-  > If you have an existing app in your repository, you may skip to the next section.
+  > If you have an existing app in your repository, you may skip to the next section. NOTE: The existing app must target .NET Core 3.1 for the pipeline to publish the app correctly.
```

Comment:
This formats the comment as an individual note element. @scubaninja, feel free to sign off once @Cmohan accepts these changes.

```suggestion
  > If you have an existing app in your repository, you may skip to the next section.
  
  > [!NOTE]
  > The application must target .NET Core 3.1 for the pipeline to succeed.
```

Thanks to you both!

Reply:
I put the specification with the note about using an existing project because you need to know that information to determine which existing project you could use. If you're using the demo project, it's already configured so the version information is nice to know but not strictly relevant. Is this a formatting thing?

Reply:
yes, it will be down to formatting in line with our other docs.

Reply:
Ok, that's fine then.